recipes/fiji-max-inscribed_circles/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "fiji-max-inscribed_circles" %}
+{% set version = "1.1.2" %}
+ 
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://maven.scijava.org/service/local/repositories/releases/content/ch/epfl/biop/MaxInscribedCircles/{{ version }}/MaxInscribedCircles-{{ version }}.jar
+  sha256: 925942f58cf45005594bb730af4dec3706793d3d2513a101b6de42984818e003
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - fiji>=20220414,!=h527b516_0,!=h9ee0642_0
+
+test:
+  files:
+    - test.groovy
+
+about:
+  home: https://imagej.net/plugins/max-inscribed-circles
+  license: GPLv3
```

Comment:
can you please add a license_file to this recipe?

Reply:
done